src/agent0/ethpy/base/transactions.py

```diff
@@ -8,6 +8,7 @@
 
 from eth_account.signers.local import LocalAccount
 from eth_typing import BlockNumber, ChecksumAddress
+from fixedpointmath import FixedPoint
```

Comment:
don't need this anymore

Reply:
removed in d21497c71bbc02a4271e9db1eb33b11e09655bc6